templates/index.html

```diff
@@ -1,51 +1,117 @@
+            href="static/leaderboard.css?v=1"
+        />
+        <link
+            type="text/css"
+            rel="stylesheet"
+            href="https://unpkg.com/bootstrap@4.4.1/dist/css/bootstrap.min.css"
+        />
+        <link
+            type="text/css"
+            rel="stylesheet"
+            href="https://unpkg.com/bootstrap-vue@2.3.0/dist/bootstrap-vue.min.css"
+        />
+    </head>
+
+    <body>
+        <nav class="navbar navbar-light bg-light">
+            <a class="navbar-brand" href="#">
+                <img
+                    src="https://www.dswc.org/wp-content/uploads/2018/07/dswc_logo_small_x1.png"
+                    width="30"
+                    height="30"
+                    class="d-inline-block align-top"
+                    alt=""
+                />
+                DSWC <span class="lead">Regatta</span>
+            </a>
+        </nav>
+
+        <div class="container-fluid my-4">
+            <div class="row">
+                <div class="col-lg-3">
+                    <div class="mb-4">
+                        <h2 class="lead">Pick a series</h2>
+                        <div id="allSeries">
+                            <b-list-group>
+                                <b-list-group-item
+                                    v-on:click="resultsPane.displaySeriesData(series)"
+                                    href="#"
+                                    v-for="series in seriesList"
+                                >
+                                    {{ series.replace('_', ' ') }}
+                                </b-list-group-item>
+                            </b-list-group>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-lg-9">
+                    <div id="resultsPane" v-show="showResults" class="mb-4">
+                        <result-table
+                            :title="summaryTitle"
+                            :tabledata="summaryData"
+                        ></result-table>
+                        <!-- <div v-for="races in raceDays">
+                        <result-table :title="summaryTitle" :tabledata="summaryData"></result-table>
+                    </div> -->
+                    </div>
+
+                    <div class="mb-4">
+                        Legend: <span class="badge badge-info">Computed</span>
+                        <span class="badge badge-warning"
+                            >Missed / No data available</span
+                        >
+                        <span class="badge badge-danger">Discarded</span>
+                    </div>
+                    <div class="mb-4">
+                        <p class="lead">Questions &amp; Comments</p>
+                        <p>
+                            If data for your race is missing or you think
+                            there's a mistake please let us know on the DSWC
+                            WhatsApp group!
```

Comment:
Can we shorten this? It looks a bit messy on my mobile. 

Maybe just say "If race data is wrong or missing let us know on the WhatsApp Group!"